.travis.yml

```diff
@@ -0,0 +1,8 @@
+sudo: false # Use new Travis container-based infrastructure
```

Comment:
I thought we coud not have Circle in a branch and travis on another, but we should be able to get something playing with protected branches.